cypress/e2e/TrialDescriptionPage/PageNotFound.feature

```diff
@@ -3,13 +3,13 @@ Feature: Page Not Found
 	Scenario: User is able to see a page not found when trying to access an invalid page
 		Given screen breakpoint is set to "desktop"
 		Given the user navigates to "/v?id=NCI-0000-00000&loc=0&rl=1"
-		Then the page title is "Page Not Found"
+		Then page title on error page is "Page Not Found"
```

Comment:
Follows how we're handling it elsewhere / handles the cypress uncaught exceptiion within the step definition rather than relying on the global exception catcher